zulipterminal/config/keys.py

```diff
@@ -154,7 +154,7 @@ class KeyBinding(TypedDict):
     },
     'ADD_REACTION': {
         'keys': [':'],
-        'help_text': 'Show/hide Emoji picker popup for current message',
+        'help_text': 'Show/hide Emoji Picker popup for current message',
```

Comment:
I'm not sure on this one, given that "Emoji Picker" is not the title used for the popup (which is different to the other cases above).

My personal preference would be to use "Emoji-picker" instead of "Emoji picker popup" and "Emoji-picker popup", but any of these three could work.

Reply:
Yes, that's a tricky one indeed.
Yep, we could remove the 'popup' word. Good one.
Googling shows a usage preference of not using a hyphen. So, I went with that.
And yes, the popup does not have a header.